projects/knora-ui/src/lib/viewer/values/link-value/link-value.component.spec.ts

```diff
@@ -123,28 +123,40 @@ describe('LinkValueComponent', () => {
       const hostCompDe = testHostFixture.debugElement;
 
       valueComponentDe = hostCompDe.query(By.directive(LinkValueComponent));
-      valueInputNativeElement = valueComponentDe.query(By.css('input')).nativeElement;
+      valueInputDebugElement = valueComponentDe.query(By.css('input.value'));
+      valueInputNativeElement = valueInputDebugElement.nativeElement;
 
       commentInputDebugElement = valueComponentDe.query(By.css('input.comment'));
       commentInputNativeElement = commentInputDebugElement.nativeElement;
 
     });
 
-    it('should display an existing value', () => {
+    it('should display an existing value', fakeAsync(() => {
 
-      expect(testHostComponent.inputValueComponent.displayValue.linkedResourceIri).toMatch('http://rdfh.ch/0001/0C-0L1kORryKzJAJxxRyRQ');
-      expect(testHostComponent.inputValueComponent.displayValue.propertyLabel).toMatch('Another thing');
```

Comment:
why have you removed this test?

Reply:
I did not remove it, I just adapted some assertions. Here, the relevant things are: Iri and label

Reply:
The label and IRI of the `linkedResource` were already checked. The test for `propertyLabel` was to check the label of the displayed value itself not the label of its `linkedResource`.